docs/source/platform_agnostic_sharing_with_kedro_viz.md

```diff
@@ -40,6 +40,10 @@ kedro>=0.18.2
 kedro viz build
 ```
 
+```{note}
+Starting from Kedro-Viz 9.1.0, previews are disabled by default for `kedro viz build`. To enable previews for all nodes of the datasets, use the `--include-preview` flag.
```

Comment:
i think this might go in the next release 9.2.0

Reply:
Okay let me update to Kedro-Viz 9.2.0

Reply:
Done.